rrtmgp/kernels-openacc/mo_gas_optics_kernels.F90

```diff
@@ -668,7 +669,7 @@ subroutine compute_Planck_source(                        &
     ! Explicitly unroll a time-consuming loop here to increase instruction-level parallelism on a GPU
     ! Helps to achieve higher bandwidth
     !
-    !$acc parallel loop collapse(3)
+    !$acc parallel loop present(planck_function) collapse(3)
```

Comment:
`planck_function` is a function. What does this clause do since the function should always be present? I thought only variables would be allowed as arguments.

Reply:
Despite the name, `planck_function` is an array :) See line 606

Reply:
My fault -- I misread the code.